org.rolisteam.rolisteam.json

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "org.rolisteam.rolisteam",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "command": "rolisteam",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--filesystem=home",
+    "--env=TMPDIR=/var/tmp",
+    "--env=GST_PLUGIN_PATH=/usr/lib/x86_64-linux-gnu/gstreamer-1.0/"
```

Comment:
This line can be dropped – the bug has been fixed since 5.12.

Reply:
ok